c++/src/kj/async-inl.h

```diff
@@ -789,16 +790,13 @@ template <typename T> Maybe<Own<T>> copyOrAddRef(Maybe<Own<T>>& t) {
   });
 }
 
-template <typename T>
-class ForkBranch final: public ForkBranchBase {
-  // A PromiseNode that implements one branch of a fork -- i.e. one of the branches that receives
-  // a const reference.
 
+template <typename T>
+class  ForkBranchBaseT: public ForkBranchBase {
```

Comment:
I decided to keep the ForBranchBase name since it is used quite a lot @harrishancock